modules/administration-guide/partials/snip_scaling-down-che.adoc

```diff
@@ -0,0 +1,6 @@
+. Scale down the {prod-short} deployment:
++
+[subs="+quotes,+attributes"]
+----
+{orch-cli} scale deployment {prod-deployment} --replicas=0 -n {prod-namespace}
```

Comment:
I believe it would be great to use some env var for namespace. Otherwise, it will work only for defaults installations, and will not work for specific setups like dogfooding / Developer Sandbox

Reply:
It make perfect sense but all documentation is based on the fact that Eclipse Che is deployed in {prod-namespace}
which is `eclipse-che` (`openshift-workspaces`)
So, to be hones I don't know how to handle this.

Reply:
I assume if namespace is different, then user must change it in the scrips manually.

Reply:
I would probably introduce NAMESPACE env var, it is really error-prone and to expect from an admin to patch the scripts manually

Reply:
I fully agree with @ibuziuk.

Reply:
so into which repo will we move the script(s) so that they can evolve outside the docs, and not be beholden to doc-related validation rules? Also then we can tell a user "set your namespace variable, curl this URL, run this script, and you're done!"

Reply:
can we use https://github.com/devfile/devworkspace-operator-docs or do we need a new repo?